pints/__init__.py

```diff
@@ -204,11 +204,20 @@ def version(formatted=False):
 from ._mcmc._differential_evolution import DifferentialEvolutionMCMC
 from ._mcmc._dram_ac import DramACMC
 from ._mcmc._dream import DreamMCMC
+from ._mcmc._dual_averaging import DualAveragingAdaption
 from ._mcmc._emcee_hammer import EmceeHammerMCMC
 from ._mcmc._haario_ac import HaarioACMC
 from ._mcmc._haario_bardenet_ac import HaarioBardenetACMC
 from ._mcmc._haario_bardenet_ac import AdaptiveCovarianceMCMC
 from ._mcmc._hamiltonian import HamiltonianMCMC
+
+if sys.hexversion >= 0x03030000:
+    from ._mcmc._nuts import NoUTurnMCMC
+else:
+    print(
```

Comment:
Should this be a warning?
If so, best use `warnings.warn` (see #1030)
If it's more of an informative piece of info, maybe use `logging` ?